lib/workers/repository/dependency-dashboard.ts

```diff
@@ -263,6 +283,15 @@ export async function ensureDependencyDashboard(
     return;
   }
   let issueBody = '';
+
+  if (configMigrationRes?.result === 'pr-exists') {
+    issueBody += `Config Migration necessary. You can view the Config Migration PR here #${configMigrationRes.prNumber}\n\n`;
```

Comment:
```suggestion
    issueBody += `Config Migration needed. See Config Migration PR: #${configMigrationRes.prNumber}\n\n`;
```

We should end the sentence with a full stop `.`. Not sure _where_ to put it in the code. :wink: